app/.snyk

```diff
@@ -1,6 +1,23 @@
 # Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
-version: v1.14.1
-ignore: {}
+version: v1.13.5
```

Comment:
Why was the version decreased?

Reply:
I wonder if Richard was running a different version of Snyk CLI than the last committer?

Reply:
Yep .. this was generated using `yarn snyk wizard` which used the snyk version from package.json. Perhaps the last committer had a more recent version installed globally?

Reply:
What is the consensus? Shall I leave the older version as is or update the version we have in package.json to match the newer (or latest) version

Reply:
I would prefer updating the snyk version as well, if possible. Either as part of this or a separate PR. It might be a simple thing to do since it seems like the newer version has been used previously.

What do you think @rBangay @twrichards ?